src/connectivities.jl

```diff
@@ -94,7 +94,7 @@ end
 Materialize a face using the `connec` list and a global vector of `points`.
 """
 materialize(connec::Connectivity{PL,N}, points::AbstractVector{P}) where {PL<:Polytope,N,P<:Point} =
-  PL(ntuple(i -> @inbounds(points[connec.indices[i]]), N))
+  PL(ntuple(i -> @inbounds(points[connec.indices[i]]), N)...)
```

Comment:
This seems unrelated to the Simplex addition. Can you please revert? Splatting leads to slow performance sometimes.